android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -274,6 +325,31 @@ public void onError(String info) {
         }
     }
 
+    /**
+     * Starts streaming a remote display to the module if there is a connected session. This method of streaming requires the device to be on API level 19 or higher
+     *
+     * @param context            a context that can be used to create the remote display
+     * @param remoteDisplayClass class object of the remote display. This class will be used to create an instance of the remote display and will be projected to the module
+     * @param parameters         streaming parameters to be used when streaming. If null is sent in, the default/optimized options will be used.
+     *                           If you are unsure about what parameters to be used it is best to just send null and let the system determine what
+     *                           works best for the currently connected module.
+     * @param encrypted         a flag of if the stream should be encrypted. Only set if you have a supplied encryption library that the module can understand.
+     * @param landscapeRange    constraints for vehicle display : aspect ratio, min/max resolutions, max diagonal size.
+     * @param portraitRange     constraints for vehicle display : aspect ratio, min/max resolutions, max diagonal size.
+     */
+    public void startRemoteDisplayStream(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass, VideoStreamingParameters parameters, final boolean encrypted, VideoStreamingRange landscapeRange, VideoStreamingRange portraitRange) {
+        Collections.addAll(listOfStreamingRanges, portraitRange, landscapeRange);
+        configureGlobalParameters(context, remoteDisplayClass, isEncrypted, listOfStreamingRanges);
+        if(majorProtocolVersion >= 5 && !internalInterface.getSystemCapabilityManager().isCapabilitySupported(SystemCapabilityType.VIDEO_STREAMING)){
+            stateMachine.transitionToState(StreamingStateMachine.ERROR);
+            return;
+        }
+        if (!HMILevel.HMI_NONE.equals(currentOnHMIStatus.getHmiLevel()) && VideoStreamManager.this.parameters == null) {
+            getVideoStreamingParams();
+        }
+        checkState();
```

Comment:
I believe these lines can be removed
```
if (!HMILevel.HMI_NONE.equals(currentOnHMIStatus.getHmiLevel()) && VideoStreamManager.this.parameters == null) {
        getVideoStreamingParams();
}
checkState();
```
When we call processCapabilitiesWithPendingStart() then the params will be setup properly based on the capabilities